lighthouse-core/config/index.js

```diff
@@ -241,19 +305,53 @@ class Config {
       configJSON = defaultConfig;
     }
 
-    this._audits = configJSON.audits ? expandAudits(
-        filterAudits(configJSON.audits, auditWhitelist)
+    this._configJSON = this._initRequirePaths(configJSON);
+
+    this._audits = this.json.audits ? expandAudits(
```

Comment:
Because we already did. The only thing I changed here was to make it a getter on this.json. So if there's a bug / invalid approach, it already existed :)